webplugin/js/app/zendesk-chat-service.js

```diff
@@ -118,6 +118,45 @@ function ZendeskChatService() {
                     console.log('zChat.sendChatMsg ', err, data);
                 }
             );
+
+            //Sending chat transcript
+            var activeConversationInfo = JSON.parse(localStorage.getItem('kommunicate')).mckActiveConversationInfo;
```

Comment:
This data is already there in some variable. see if we can use the same.

Reply:
Done